Mathlib/Tactic/IntervalCases.lean

```diff
@@ -0,0 +1,282 @@
+/-
+Copyright (c) 2022 Mario Carneiro. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro, Heather Macbeth
```

Comment:
I have never been involved with this tactic, maybe this is copy-pasted from another header?